build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/docstring.rb

```diff
@@ -118,10 +118,6 @@ def html_to_markdown(html, options = {})
         if html
           html = "<p>#{html}</p>" unless html.match(/<\w+>/)
 
-          # unescaped curly braces cause YARD errors, they are interpreted
-          # as code links.
-          html = html.gsub('{', "\\{").gsub('}', "\\}")
```

Comment:
could we take an `escape_curly_braces` option and pass it only when this is called for code docs?

Reply:
That doesn't fix inline code snippets in the documentation. This applies on the whole block of documentation text.